docs/source/user/interface.rst

```diff
@@ -6,14 +8,16 @@ exploratory computing. While JupyterLab has many features found in
 traditional integrated development environments (IDEs), it remains
 focused on interactive, exploratory computing.
 
-The JupyterLab interface consists of a main work area containing tabs of
-documents and activities, a collapsible left sidebar, and a menu bar.
-The left sidebar contains a file browser, the list of running kernels
-and terminals, the command palette, the notebook cell tools inspector,
-and the tabs list.
+The JupyterLab interface consists of a :ref:`main work area <main_area>` containing tabs of
+documents and activities, a collapsible :ref:`left sidebar <left_sidebar>`, and a :ref:`menu bar <menu_bar>`.
+The left sidebar contains a :ref:`file browser <working_with_files>`, the :ref:`list of running kernels
+and terminals <running>`, the :ref:`command palette <commands>`, the :ref:`notebook cell tools inspector <notebook>`,
```

Comment:
The cell inspector tool doesn't have a section, so probably shouldn't have a reference?

Reply:
I was referring to this section in that link:

>You can connect a :ref:`code console <code_console>` to a notebook kernel to have a log of
>computations done in the kernel, in the order in which they were done.
>The attached code console also provides a place to interactively inspect
>kernel state without changing the notebook. Right-click on a notebook
>and select “Create Console for Notebook”:

I can write it it differently if you think that'd be better?

Reply:
I also included a link to the notebook section in getting started instead if you'd rather have the link there @jasongrout

Reply:
My point is that we're not talking about a console connected to the notebook here; we're talking about the notebook cell tools inspector in the left sidebar in this text. That's not talked about in the notebook section, IIRC, so it's confusing to me to link to the notebook page.

On the other hand, perhaps we should just add a short section to the notebook file mentioning the cell inspector tools.